scripts/session/session-logout.sh

```diff
@@ -1,8 +1,21 @@
-#!/bin/sh
+#!/usr/bin/sh
 #
 # name: Log Out
 # icon: system-log-out
 # description: Log out to the login screen
 # keywords: log out logout
 
-gnome-session-quit --logout
+set -eu
+
+is_gnome() {
+  [ "${XDG_CURRENT_DESKTOP:-}" = "GNOME" ] && \
```

Comment:
This fails on Pop!_OS because XDG_CURRENT_DESKTOP is `Unity`. Only the dbus command is required to check if gnome-shell is running.